oak_ml_transparency/mnist/README.md

```diff
@@ -0,0 +1,109 @@
+# The MNIST example
+
+This folder contains scripts for building an MNIST model and evaluating it
+against adversarial examples. The build script is only included here to make the
+example self-contained. The eval script should work with any MNIST model in
+[Tensorflow SavedModel format](https://www.tensorflow.org/guide/saved_model).
+
+## Building the Docker image
+
+We use a Docker image, with all the required dependencies installed to build the
+model, and run the evaluation. Having such a Docker image is necessary for
+achieving transparency. The Dockerfile should include the script for running the
+evaluation. More specifically, with the following lines in the Dockerfile, we
+embed the evaluation script in the Docker image:
+
+```bash
+RUN mkdir /project
+WORKDIR /project
+ADD . /project
+```
+
+Ideally, and as is the case in this example, the Dockerfile and the evaluation
+script should belong to the same repository, so that a single Git commit hash
+can uniquely specify both the content of the Dockerfile, and the script.
+
+Build the Docker image using the following command:
+
+```shell
+cd oak_ml_transparency/mnist
+docker build -t mnist-eval .
+```
+
+Note that if you make any changes to the script, you have to rebuild the Docker
+image for the changes to take effect.
+
+## Run the image
+
+For this example, we run the Docker image in the interactive mode:
+
+```shell
+$ docker run --tty --interactive --rm \
+  --network=host \
+  mnist-eval bash
+```
+
+This gives us access to a shell:
+
+```bash
+==========
+== CUDA ==
+==========
+
+CUDA Version 11.3.1
+
+Container image Copyright (c) 2016-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+
+This container image and its contents are governed by the NVIDIA Deep Learning Container License.
+By pulling and using the container, you accept the terms and conditions of this license:
+https://developer.nvidia.com/ngc/nvidia-deep-learning-container-license
+
+A copy of this license is made available in this container at /NGC-DL-CONTAINER-LICENSE for your convenience.
+
+WARNING: The NVIDIA Driver was not detected.  GPU functionality will not be available.
+   Use the NVIDIA Container Toolkit to start this container with GPU support; see
+   https://docs.nvidia.com/datacenter/cloud-native/ .
+
+root@hostname:/project#
+```
+
+Inspect the directory structure using the `ls` command to make sure the
+`eval.py` script is there:
+
+```shell
+root@hostname:/project# ls
+Dockerfile  README.md  build.py  eval.py  mnist-model
+```
```

Comment:
This could be of type `console`

```suggestion
```console
$ root@hostname:/project# ls
Dockerfile  README.md  build.py  eval.py  mnist-model
```
```

Reply:
Thanks. Done.